webapp/src/actions.js

```diff
@@ -7,7 +7,7 @@ import {
 } from './action-types';
 
 export const getCourses = () => createAction({
-  endpoint: `${ENDPOINT}/api/courses/?limit=11000`,
+  endpoint: `${ENDPOINT}/api/courses/?limit=12000`,
```

Comment:
We could just set `limit=1000000` until we figure out #25 if you want, although I don't think it matters too much.

Reply:
Strangely, it does matter in the browser. Times go up by ~400ms. CLI, with curl, experiences no such odd changes.

Reply:
🤔 it looks pretty consistent to me:
![Screen Shot 2020-05-26 at 6 26 00 PM](https://user-images.githubusercontent.com/7410405/82959938-a455bf00-9f7e-11ea-8397-fac33405f9cd.png)

Again, doesn't really matter if you just wanna get this merged in tonight but I dunno why that'd be happening for you.

Reply:
Weird, anyway for now I'm content to just update it to the next highest number.